ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/DX9surface.cpp

```diff
@@ -41,6 +41,14 @@ namespace enigma {
 vector<Surface*> Surfaces(0);
 D3DCOLOR get_currentcolor();
 
+void surface_copy_to_systemmem(IDirect3DSurface9 **src, IDirect3DSurface9 **dest) {
```

Comment:
Took me a bit to figure out what was going on. "System memory" isn't really a term; "main memory" would be fine, but even just maintaining the distinction between "RAM" and "VRAM" would suffice. Personally, I think just `surface_copy_to_memory` (or ram) is fine.

Reply:
Ok: b40dd33ed5058363a1a9ec115de6ce3d7de5ec69

So I went with that just to be consistent with the `D3DPOOL_SYSTEMMEM` memory pool. However, I realized I hated both typing and reading it. Then I decided to change it because I realized that you are correct, Microsoft is just trying to confuse people.